plugins/hypervisors/xenserver/src/com/cloud/hypervisor/xenserver/resource/CitrixResourceBase.java

```diff
@@ -164,9 +165,12 @@
  *
  */
 public abstract class CitrixResourceBase implements ServerResource, HypervisorResource, VirtualRouterDeployer {
-
+    /**
+     * RELVMOISCSI = used for resigning metadata (like SR UUID and VDI UUID when a
+     * particular storage manager is installed on a XenServer host (for back-end snapshots to work))
+     */
```

Comment:
I would put the value javadoc on the value and describe only the whole enum here. (PR coming to that end on another enum, shortly)